.github/workflows/docstring_check.yaml

```diff
@@ -0,0 +1,19 @@
+name: docstring-check
+on:
+    push:
+
+jobs:
+  build:
+    name: docstring-check
+    runs-on: ubuntu-18.04
+    steps:
+        - name: Fetch repository
+          uses: actions/checkout@v2.7.1
```

Comment:
What version use here? I thought the latest MorphIO `v2.7.1` but seems this version is about something else?

Reply:
This is the version of the `checkout action`:
https://github.com/actions/checkout

So nothing related to morphio.

Reply:
Thank you so much! Lazy me